packages/gui/package.json

```diff
@@ -2,13 +2,95 @@
   "name": "@chia/gui",
   "author": "Chia Network <hello@chia.net> (https://chia.net/)",
   "description": "GUI for Chia Blockchain",
+  "license": "Apache-2.0",
   "productName": "Chia Blockchain",
   "private": true,
   "homepage": "./",
   "main": "build/electron/main.js",
   "engines": {
     "node": ">=12.20.0"
   },
+  "build": {
+    "productName": "chia",
```

Comment:
Is there a reason we shouldn't capitalize it as "Chia"? The macOS PoC build has the app named "chia.app" when it should be "Chia.app"

Reply:
@wallentx 
Any ideas? I cherry-picked the change from this commit (https://github.com/Chia-Network/chia-blockchain-gui/pull/903/commits/7077750fe6fc532c7a17e08dd47bfcf99b15be73)

Reply:
Ah ok, maybe this was done for Linux, but on macOS/Windows we should capitalize it. We can address that in the build script I think.

Reply:
Updated mac executable file name to `Chia.app` from `chia.app`.